app/models/incoming_email.rb

```diff
@@ -42,8 +42,4 @@ def body
   def datetime
     received_at
   end
-
```

Comment:
This method removal to be causing a test failure. Consider restoring the method but giving it a null guard on `client.intake`.